.github/workflows/test.yml

```diff
@@ -25,9 +25,10 @@ jobs:
             ruby: truffleruby
           - os: windows-latest
             ruby: truffleruby-head
-          # Remove it once https://github.com/ruby/setup-ruby/issues/680 is fixed
+        include:
           - os: windows-latest
-            ruby: 3.4
+            ruby: mswin
```

Comment:
`ruby` -> `os` order will be better to align other job names:

```yaml
- ruby: mswin
  os: windows-latest
```

https://github.com/ruby/rdoc/actions/runs/12793372976/job/35666070365?pr=1277 : "test (3.0, ubuntu-latest)"
https://github.com/ruby/rdoc/actions/runs/12793372976/job/35666079817?pr=1277 : "test (windows-latest, mswin)"

Another approach: Add `versions: mswin` to `ruby-versions` and add `os: ubuntu-latest, ruby: mswin`/`os: macos-latest, ruby: mswin` excludes.

Reply:
I updated to use the  `versions` input, I believe it wants a JSON array...